client-core/src/logo.rs

```diff
@@ -0,0 +1,112 @@
+        "Old bankers never die, they\njust... pass the buck",
+        "I won $3M on the lottery so I donated a quarter\nof it to charity. Now I have $2,999,999.75.",
+        "There are two rules for financial success:\n1) Don't tell everything you know.",
+        "Top tip: never say \"oops\", instead\nalways say \"Ah, Interesting!\"",
+        "Computers are useless. They can only\ngive you answers.  -- Picasso",
+        "Regular naps prevent old age, especially\nif you take them whilst driving.",
+        "Always borrow money from a pessimist.\nHe won't expect it back.",
+        "War does not determine who is right.\nIt determines who is left.",
+        "A bus stops at a bus station. A train stops at a\ntrain station. What happens at a workstation?",
+        "I got a universal remote control yesterday.\nI thought, this changes everything.",
+        "Did you ever walk into an office and\nthink, whiteboards are remarkable!",
+        "The good thing about lending out your time machine\nis that you basically get it back immediately.",
+        "I used to work in a shoe recycling\nshop. It was sole destroying.",
+        "What did the fish say\nwhen he hit a wall? Dam.",
+        "You should really try a seafood diet.\nIt's easy: you see food and eat it.",
+        "I recently sold my vacuum cleaner,\nall it was doing was gathering dust.",
+        "My professor accused me of plagiarism.\nHis words, not mine!",
+        "Change is inevitable, except\nfrom a vending machine.",
+        "If at first you don't succeed, destroy\nall the evidence that you tried.",
+        "If at first you don't succeed, \nthen we have something in common!",
+        "Moses had the first tablet that\ncould connect to the cloud.",
+        "How did my parents fight boredom before the internet?\nI asked my 17 siblings and they didn't know either.",
+        "Cats spend two thirds of their lives sleeping\nand the other third making viral videos.",
+        "The problem with troubleshooting\nis that trouble shoots back.",
+        "I named my dog 'Six Miles' so I can tell\npeople I walk Six Miles every day.",
+        "People used to laugh at me when I said I wanted\nto be a comedian. Well they're not laughing now!",
+        "My wife just found out I replaced our bed\nwith a trampoline; she hit the roof.",
+        "My boss asked me who is the stupid one, me or him?\nI said everyone knows he doesn't hire stupid people.",
+        "Don't trust atoms.\nThey make up everything.",
+        "Keep the dream alive:\nhit the snooze button.",
+        "Rest in peace, boiled water.\nYou will be mist.",
+        "When I discovered my toaster wasn't\nwaterproof, I was shocked.",
+        "Where do cryptographers go for\nentertainment? The security theatre.",
+        "How did the Java programmer get rich?\nThey inherited a factory.",
+        "Why did the developer quit his job?\nHe didn't get ar-rays."
+    ];
```

Comment:
taking other project's jokes isn't very original -- especially if they refer to the other project ("pacman logo"), they don't make sense

Reply:
better to be removed or you can come up with original jokes

Reply:
Ok

Reply:
@leejw51crypto Can help ;)